ci/run_tests_test.go

```diff
@@ -50,7 +50,12 @@ func TestRunTests(t *testing.T) {
 		JobOwner: "muggles",
 	}
 
-	ed, err := runTests("scripts", &Docker{}, info, runData)
+	runner, err := NewDockerCI()
+	if err != nil {
+		t.Fatalf("failed to set up docker client: %v", err)
```

Comment:
Just err. Don’t need the extra details in the tests... the err will tell the same anyway

Reply:
Fixed.